src/test/java/com/amazon/opendistroforelasticsearch/sql/unittest/parser/SqlParserTest.java

```diff
@@ -1116,6 +1116,36 @@ public void caseWhenTestWithouhtElseExpr() throws SqlParseException {
 
     }
 
+    @Test
+    public void caseWhenSwitchTest() throws SqlParseException {
+        String query = "SELECT CASE weather "
```

Comment:
Small thing- but increasing the indent so that all the lines of the query are inline would improve readability

Reply:
Cool, will do it. Thanks!